doc/source/options.rst

```diff
@@ -402,6 +402,10 @@ display.html.table_schema               False        Whether to publish a Table
 display.html.border                     1            A ``border=value`` attribute is
                                                      inserted in the ``<table>`` tag
                                                      for the DataFrame HTML repr.
+display.html.use_mathjax                True         When True, IPython notebook will process 
```

Comment:
IPython -> Jupyter

Reply:
I'll change this, but just want to let you know that I used IPython instead of Jupyter for consistency with the documentation for other options. Perhaps these also need updating?

Reply:
Sure, that'd be good to correct them all.

Reply:
There are 148 matches across 41 files for `IPython` (case sensitive). Not all of these need changing – it's quite nuanced. This is probably better left for a separate PR.

With regards to this PR, shall I use Jupyter or IPython when documenting this new option?